jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt

```diff
@@ -42,6 +46,15 @@ abstract class AwsReuseSolutionTestBase : BaseTestWithSolutionBase() {
         allowCustomDotnetRoots()
     }
 
+    @BeforeClass
+    fun setUpBuildToolPath() {
+        if (SystemInfo.isWindows) {
+            PrepareTestEnvironment.dotnetCoreCliPath = "C:\\Program Files\\dotnet\\dotnet.exe"
+            setUpDotNetCoreCliPath(PrepareTestEnvironment.dotnetCoreCliPath)
+            setUpCustomToolset(msBuild)
+        }
+    }
+
```

Comment:
are these still needed?

Reply:
Yes, otherwise it likes to pick the wrong version, we can probably kill all these off when we go 202 min